src/Core/Enums/LayoutArea.cs

```diff
@@ -15,7 +16,8 @@ public enum LayoutArea
     Footer,
 
     /// <summary />
-    Menu,
+    [Description("nav")]
```

Comment:
Why do you need this attribute?

Reply:
So navigation can be shortened to nav in CSS (AddGridAreaStyles is using the ToAttributeValue to get the name from the enum). Think in CSS it is better to use 'nav' instead of 'navigation'

Reply:
OK. Can you add the Description attribute on the other enum values (to be coherent)?

And including an empty line between each items.